docs/modules/ROOT/pages/documentation/payara-server/asadmin-commands/Enable-JLine-Event-Expansion.adoc

```diff
@@ -0,0 +1,42 @@
+= Enable JLine Event Expansion
+
+Enable this feature if you would like to use the Event Designators inside the Asadmin Multimode.
+
+An Event Designator is a reference to a command line entry in the history list.
```

Comment:
We can add a link for more info about event expansion:
```suggestion
An [Event Designator](https://www.gnu.org/software/bash/manual/html_node/Event-Designators.html) is a reference to a command line entry in the history list.
```
I wanted to suggest this also for the community PR but forgot. I'll do the change in the community docs myself.